contracts/ParametersBatchUpdater.sol

```diff
@@ -5,35 +5,124 @@
 */
 pragma solidity ^0.7.1;
 
-import "./vault-managers/VaultManagerParameters.sol";
-import "./Vault.sol";
-import "./CollateralRegistry.sol";
-import "./oracles/BearingAssetOracleSimple.sol";
+
+interface IVaultManagerParameters {
```

Comment:
It makes sense to factor them out as standalone files in `contracts/interfaces` subfolder and reuse (copy-pasted interfaces are still copy-paste).
Also it's better not to omit parameter names.